content/docs/contributing/policy.md

```diff
@@ -3,59 +3,125 @@ title: Feature Policy
 description: 'cert-manager contributing guide: Contribution Policy'
 ---
 
-We are open to feature requests and PRs implementing those. If you plan on contributing a new feature into cert-manager we recommend creating an issue first for it to be discussed with the cert-manager maintainers. Another possibility is bringing it up in a community meeting discussion for an open discussion with the maintainers and community on the implementation.
+We love to receive both feature requests and PRs which add stuff to cert-manager; the community is at the heart of what we do!
 
-# Features we will not allow
+If you're thinking of adding a feature, we recommend you read this doc to maximize the chances of your contribution getting the attention it deserves and hopefully to get it merged quickly!
 
-Certain features have been requested before, but have not been implemented properly or not able to be implemented without breaking the security model.
+We recommend creating an issue first for it to be discussed with the cert-manager maintainers. Another possibility is bringing it up in a community meeting discussion for an open discussion with the maintainers and community on the implementation.
```

Comment:
```suggestion
We recommend creating an issue first for it to be discussed with the cert-manager maintainers. Another possibility is bringing it up in a community meeting for an open discussion with the maintainers and community on the implementation.
```

Repeating the word discussion so close together seems clumsy. Maybe repeating community is also unnecessary?

Reply:
Yeah this was definitely an editing mistake on my part, good catch :+1: Fixed!